src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/MainActivity.cs

```diff
@@ -9,7 +9,7 @@ namespace Uno.Material.Samples.Droid
 	[Activity(
 			MainLauncher = true,
 			ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize,
-			WindowSoftInputMode = SoftInput.AdjustPan | SoftInput.StateHidden
+			WindowSoftInputMode = SoftInput.AdjustResize | SoftInput.StateHidden
```

Comment:
I thought we usually use AdjustNothing in uno apps

Reply:
Oh, right.